tests/unit/server/bundle_manager/schedule_run_bundles_test.py

```diff
@@ -92,3 +92,27 @@ def test_finalizing_bundle_gets_finished(self):
 
         bundle = self.bundle_manager._model.get_bundle(bundle.uuid)
         self.assertEqual(bundle.state, State.READY)
+        self.assertNotEqual(
```

Comment:
In fact, I think here, we can actually use `assertEqual` and ensure it's equal to the proper values, since the start / end times are mocked anyway (so in this case, it would be 5)

Reply:
makes sense to me

Reply:
@adiprerepa can you address this comment when you get a chance?

Reply:
done